src/Core/SecretsManager/Repositories/Noop/NoopServiceAccountRepository.cs

```diff
@@ -56,4 +56,9 @@ public Task<int> GetServiceAccountCountByOrganizationIdAsync(Guid organizationId
     {
         return Task.FromResult(0);
     }
+
+    public Task<IEnumerable<ServiceAccount>> GetManyByOrganizationIdAsync(Guid organizationId)
```

Comment:
I don't think we need this anymore?

We have `GetManyByOrganizationIdAsync(Guid organizationId, Guid userId, AccessClientType accessType)` at line 8 which covers the interface.